starrocks-stream-load-sdk/src/main/java/com/starrocks/data/load/stream/properties/StreamLoadTableProperties.java

```diff
@@ -114,6 +118,20 @@ private Builder() {
 
         }
 
+        // This function does not copy the uniqueKey and properties attributes because the uniqueKey 
+        // is generated in the StreamLoadTableProperties constructor, and the properties are automatically 
+        // populated during the build process.
+        public Builder copyFrom(StreamLoadTableProperties streamLoadTableProperties) {
+            database(streamLoadTableProperties.getDatabase());
+            table(streamLoadTableProperties.getTable());
+            columns(streamLoadTableProperties.getColumns());
+            enableUpsertDelete(streamLoadTableProperties.isEnableUpsertDelete());
```

Comment:
`columns` and `upsertDelete` may be different for different tables, so it's better not to copy them?